src/vos/vos_tree.c

```diff
@@ -468,15 +460,15 @@ svt_rec_store(struct btr_instance *tins, struct btr_record *rec,
  */
 static int
 svt_rec_load(struct btr_instance *tins, struct btr_record *rec,
-	     struct vos_key_bundle *kbund, struct vos_rec_bundle *rbund)
+	     struct vos_svt_key *key, struct vos_rec_bundle *rbund)
 {
-	struct svt_hkey		*skey = (struct svt_hkey *)&rec->rec_hkey[0];
+	struct vos_svt_key	*skey = (struct vos_svt_key *)&rec->rec_hkey[0];
```

Comment:
The key of the "btr_record" only contains the 64-bits main epoch, right? So the "key->sk_minor_epc" need another assignment.

Reply:
ah, I probably missed this one when I changed it :-)